loopFreq    : 1
order       : 4 
orderMin    : 3

